expense/src/execute/approve.ts

```diff
@@ -12,7 +12,7 @@ async function run() {
   await handle.start(expenseId);
 
   await new Promise((resolve) => setTimeout(resolve, 50));
-  await handle.signal.approve();
+  await handle.signal(approveSignal);
```

Comment:
If the workflow does not support this signal, type-checking won't catch it here, right? Will it be caught at runtime?

Reply:
no, it will not be caught at runtime, and the signal will just silently be recorded with no listener. Roey and I have considered printing a warning when signals are sent with no listener, but i am unclear if this should be implemented at the SDK or Server level.